### DISCUSSION TITLE
Erro no build de app Flutter no MacBook

### DISCUSSION THREAD

**Discussion by dmachadoq** [Flutterando/forum#203] [Perguntas e Respostas]
Ao executar o build do app, aparece o seguinte: 

Error (Xcode): Build input file cannot be found:
'/Users/davidmachado/Documents/dev/flutter-apps/encontrodasaguas/ios/Runner/Base.lproj/Ma
in.storyboard'. Did you forget to declare this file as an output of a script phase or
custom build rule which produces it?

Error (Xcode): Build input file cannot be found:
'/Users/davidmachado/Documents/dev/flutter-apps/encontrodasaguas/ios/Runner/Base.lproj/La
unchScreen.storyboard'. Did you forget to declare this file as an output of a script
phase or custom build rule which produces it?

Aparece também:

aparece tb esse erro: Command PhaseScriptExecution failed with a nonzero exit code

Quem pode, por favor, me ajudar?

**Comment by eduardoflorence** [comment]
Parece ter corrompido a sua pasta `ios`, então seria melhor recriá-la.
Se você não fez nenhuma configuração especial par ao ios, basta excluir esta pasta e executar o comando abaixo **dentro da pasta do seu projeto**:

```terminal
flutter create .
```
_Obs: Atente-se para o "ponto final" após o create_

**Comment by dmachadoq** [comment]
Recriei. E aparentemente houve um avanço.. mas ainda aparece esse erro:

[!] CocoaPods did not set the base configuration of your project because your project already has a custom config set. In order for CocoaPods integration to work at all, please either set the base configurations of the target Runner to Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig or include the Target Support Files/Pods-Runner/Pods-Runner.profile.xcconfig in your build configuration (`Flutter/Release.xcconfig`).
